plugins/afl-scoreboard/manager.py

```
@@ -0,0 +1,961 @@
+            super().on_config_change(new_config)
+        else:
+            self.config = new_config or {}
+
+        self.is_enabled = self.config.get("enabled", True)
+        self.display_duration = float(self.config.get("display_duration", 30))
+        self.game_display_duration = float(self.config.get("game_display_duration", 15))
+        self.live_priority = bool(self.config.get("live_priority", False))
+
+        with self._config_lock:
+            # Drain in-flight update threads before replacing managers
+            for name, thread in list(self._active_update_threads.items()):
```

Ruff (0.15.21)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)